articles/flows/guides/device-auth/includes/receive-tokens.md

```diff
@@ -0,0 +1,26 @@
+
+## Receive Tokens
+
+While the user has been authenticating and authorizing the device, the device app has continued to poll the token URL to request an Access Token. 
+
+Once the user has suucessfully authorized the device, you'll receive an `HTTP 200` response with a payload containing `access_token`, `refresh_token`, and `token_type` values:
```

Comment:
optionally it could get an `id_token` too.

Reply:
I changed the "request device code" section ("scope" parameter in table) and added that you need to add openid to get the ID Token. Between that and the (optionally) for ID Token and Access Token, I think this is done.